sandbox.go

```diff
@@ -691,6 +691,9 @@ func (sb *sandbox) EnableService() error {
 func (sb *sandbox) DisableService() error {
 	logrus.Debugf("DisableService %s START", sb.containerID)
 	for _, ep := range sb.getConnectedEndpoints() {
+		if err := ep.deleteServiceInfoFromCluster(sb, "DisableService"); err != nil {
```

Comment:
@sanimej I'm not saying is wrong. I'm saying that we have to test it properly for all the other cases that we work in the past week to be sure to not introduce again races here.